src/org/usfirst/frc4904/robot/commands/IndexerUnload.java

```diff
@@ -0,0 +1,12 @@
+package org.usfirst.frc4904.robot.commands;
+
+
+import org.usfirst.frc4904.robot.RobotMap;
+import org.usfirst.frc4904.robot.subsystems.Shooter;
+import org.usfirst.frc4904.standard.commands.motor.MotorConstant;
+
+public class IndexerUnload extends MotorConstant {
+	public IndexerUnload() {
+		super(RobotMap.Component.shooter.indexer, Shooter.INDEXER_UNLOAD_SPEED);
```

Comment:
Why do we use RobotMap.Component.shooter.indexer here and the flywheel directly elsewhere?